### PR TITLE
Minor fixes in the motivation page.

### PR DESCRIPTION
**Pull request by ahadas**
Really minor fixes.
Signed-off-by: Arik Hadas <ahadas@redhat.com>

### CODE REVIEW

**Comment by rmohr** [comment]
Looks good to me, but someone with more proven English skills has to look at that. :+1:

File: about.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ layout: default
 ## Motivation
 
 There is a wide range of management applications dealing with different
-aspects of operating system and workload virtualization, i.e. oVirt (data
+aspects of operating system and workload virtualization, i.e., oVirt (data
```
Comment (stu-gott) [inline about.md]: This line was incorrect before it was tweaked: "i.e." comes from latin meaning "that is". This basically means we're declaring the items we list afterwards to be the definitive list of what makes up the entire universe of use cases. The original intent probably was to use the abbreviation "e.g." which means "for example".

Also the actual change on this line (to introduce a comma) is incorrect. You can tell this by reading aloud the words ''for example" in place of "e.g."

File: about.md [hunk 2]
```diff
@@ -23,15 +23,15 @@ between [oVirt](https://ovirt.org) and [OpenStack](https://openstack.org).
...
+  particular type of workload they wish to run.
 
 The Kubernetes container runtime(s) reliance on a shared kernel, provides
 insufficient security isolation for certain deployment scenarios. Administrators
 of such deployments may wish to be able to use hardware virtualization to
 strongly separate untrusted workloads.
 
 Users with existing applications may not be in a position to adopt the
-application container model straightaway. Currently they have to continue using
+application container model straightaway. Currently, they have to continue using
```
Comment (stu-gott) [inline about.md]: This comma is allowed in English but probably entirely unnecessary as it presence or absence is irrelevant.

File: about.md (hunk 1 above)
Comment (ahadas) [inline about.md]: right, change to e.g.
I don't know, I always see a comma after i.e. and e.g. - so it looked kind of awkward to me, but ok.

**Pull request merged**